app/components/availability_component.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
```

Comment:
Please remove this comment.

Reply:
No problem. 

I was going to suggest that this magic comment improves performance (https://www.mikeperham.com/2018/02/28/ruby-optimization-with-one-magic-comment/), but that does not always appear to be the case according to this post: https://accidentaltechnologist.com/ruby-on-rails/frozen_string_literal-the-not-so-magical-comment/

Reply:
I seem to remember that it's not needed with Ruby 3.0 either.